### ISSUE TITLE
missing postprocessing config

### ISSUE THREAD

**Issue by ChuangMRILab** [huangnick/mcPVS-Net#1]
Hi, the pre-trained model worked fine with nnUNet_predict. However, it couldn't generate the PVS mask due to the lack of postprocessing setting. Please help to provide the postprocessing.json file in the model. Thank you.

**Comment by mohamadelsh** [comment]
> Hi, the pre-trained model worked fine with nnUNet_predict. However, it couldn't generate the PVS mask due to the lack of postprocessing setting. Please help to provide the postprocessing.json file in the model. Thank you.

I ran into the same issue.

**Comment by huangnick** [comment]
Hi, the automatic test by nnU-Net suggested that post-processing did not improve accuracy. Therefore, there is no need to run post-processing.

**Comment by mohamadelsh** [comment]
Thank you for your response. I misread the opening comment - I actually could not run the pre-trained model, as nnUNet_predict is asking for a 'plans.json' file which is unavailable with the pretrained model.
